package.json

```diff
@@ -29,7 +29,6 @@
   },
   "dependencies": {
     "@apmg/enceladus": "^0.5.4",
-    "@apmg/mimas": "^0.3.2",
```

Comment:
I don't know enough about amat and mimas to know the answer: will moving the mimas dependency to devDependencies break things in production?